### PR TITLE
Mobile navigaton menu

### PR DESCRIPTION
**Pull request by Agetuni**
- Add a navigation menu 
- Make it interactive using CSS
- add links to navigate to  th sections

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Nice job fixing your linter errors

File: style.css [hunk 1]
```diff
@@ -17,7 +21,39 @@
...
+  .mobile_nav {
+    background-color: #191765;
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+  }
+
+  .mobile_menu {
```
Comment (topeogunleye) [inline style.css:42]: - [ ] I kindly suggest that you follow the Mobile Design by adding the purple arrows on the right side of the nav menu links.
- [ ] I kindly suggest you reduce the margin-top of `.mobile_menu` to make it match the Figma template design

| Your Design  | Template Design |
| ------------- | ------------- |
| image  | Screenshot from 2022-07-19 16-37-54  |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @Agetuni,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -16,11 +17,24 @@
...
         </div>
         <nav class="navigation">
-            <i class="fa-solid fa-bars mobile_element "></i>
+            <div class="mobile_nav show_hide" id="modile_nav">
+                <div class="mobile_nav_close" id="close_mobile_humberger">
+                    <img src="Images/Icon.png" alt=" close menu" class=" mobile_element">
+                </div>
+                <ul class="mobile_menu mobile_element">
+                    <li class="mobile_nav_items"><a href="#">Portoflio</a><img src="Images/icons/rightarrow.png" alt=" Go to Portoflio section"> </li>
+                    <li class="mobile_nav_items" ><a href="#about">About</a><img src="Images/icons/rightarrow.png" alt=" Go to about section"></li>
```
Comment (danmainah) [inline index.html:26]: - [ ] [optional] Kindly ensure your portfolio and about link , take the user to the correct section of the page, eg about to the about section.

**Review by danmainah — APPROVED** [review]
Hi @Agetuni and @mohamed20163858 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**